package.json

```diff
@@ -411,6 +411,7 @@
     "Sulabh Bista <sul4bh@gmail.com>",
     "Sune Simonsen <sune@we-knowhow.dk>",
     "Svetlana <39729453+Lana-Light@users.noreply.github.com>",
+    "Sylvain Stéphant <sstephant+github@gmail.com>",
```

Comment:
Remove this change. There's a script that handles this...

Reply:
unless it matches *exactly*, the name will be duplicated.

Reply:
Ok I did not know this.